docs/env-variables.md

```diff
@@ -0,0 +1,35 @@
+# Different Environment Variables used
+
+This page describes about the different environment variables that are
+used to configure kube-scheduler-simulator. 
+
+Please refer [docker-compose.yml](./docker-compose.yml) as an example
+use.
+
+`Port`: (required) This is the port number on which kube-scheduler-simulator
+server is started. It's default value is `1212`.
+
+`EtcdURL`: (required) This is the URL for etcd. The simulator runs kube-apiserver
+internally, and the kube-apiserver uses this etcd. It's default value is `http://simulator-etcd:2379`.
```

Comment:
No default value on it. And I guess you mean `KUBE_SCHEDULER_SIMULATOR_ETCD_URL`.

```suggestion
`KUBE_SCHEDULER_SIMULATOR_ETCD_URL`: (required) This is the URL for etcd. The simulator runs kube-apiserver
internally, and the kube-apiserver uses this etcd.
```

Reply:
Updated.